data/rulei/rulei.txt

```diff
@@ -0,0 +1,86 @@
+			choice
+				`	"Hello?"`
+				`	"Is there someone there?"`
+				`	"Show yourself!"`
+			scene "scene/rulei flash"
+			`	"Human."`
+			`	An image suddenly flashes in your head as you feel millions of eyes staring at you from beyond the boundaries of space. Looking around again, you see a strange-looking spacecraft a bit off into the distance in front of you facing your direction. The voices continue to surround your mind with only slightly reduced intensity.`
+			choice
+				`	"Who are you?"`
+				`	"Where am I?"`
+					goto where
+			`	Numerous voices start up, but rather than becoming louder, they go back and forth in your head, as though they are having a discussion. After a few seconds, they fall silent, then speak a single word.`
+			`	"Precursors."`
+			choice
+				`	"What is this place?"`
+				`	"Precursors of what?"`
+			label where
+			`	Once again, the whispers start to pick up with renewed strength. The swirl of endless vocals assaults your head, each moment more agonizing than the last. Finally, they cease, and you are left in silence.`
+			`	"An End."`
+			choice
+				`	"An end to what?"'`
+				`	"What do you mean by that?"`
+			`	The voices resume, but you sense a newfound dreariness in their tone. You try once again to communicate with the ship, but despite constant whispering in the background, it no longer answers.`
+			`Taking another look at the vessel, it seems to have some sort of advanced origin. Eerie star-like patterns flow on its surface, and it doesn't visibly have any standard means of propulsion. At the rear end is some type of floating ring of objects that rotate in a pecurliar manner. The Rulei were probably far more advanced than anything human, even possibly in line with the Quarg.`
+			`The planets in this system seem to have an abnormal appearance similar to that of the starship, but for some reason even looking at them gives you a headache. Something unusual must have happened here a long time ago.`
+				decline
+				
+				
+mission "Rulei Planet"
+	invisible
+	landing
+	source
+		attributes "rulei planet"
+	on offer
+		conversation
+			`	It feels as though there is a tremendous weight on your mind while you visit this world. The same sort of purple starlit hues of the alien ship from above can also be found in various strange formations across the land. Unlike the the whispers from before, the voices here are painfully unbearable, especially the closer you get to the surface.`
+			`You can barely hear yourself think, and it's enough to drive any living thing mad. However, the most ominous sensation here is a persistent feeling of your consciousness being pulled at from beneath you, like something is attempting to drag your mind into the planet below.`
+		
+			`It's best you don't remain here for too long.`
```

Comment:
What's with the inverted spacing? I imagine it was a mistake, but it has me thinking that we could stylize this strangely to fit the mood.
```suggestion
			`It feels as though there is a tremendous weight on your mind while you visit this world. The same sort of purple starlit hues of the alien ship from above can also be found in various strange formations across the land. Unlike the whispers from before, the voices here are painfully unbearable, especially the closer you get to the surface.`
			`	You can barely hear yourself think.`
			`											It's enough to drive any living thing mad.`
			`	The most ominous sensation here is a persistent feeling of your consciousness being pulled at from beneath you.
			`	Like something is attempting to drag`
			`																					your`
			`																							mind into the planet below.`
			``
			``
			``
			`	It's best you don't remain here for too long.`
```
![image](https://user-images.githubusercontent.com/17688683/215307210-a3dac4fd-9b13-4abd-aaed-d8e560d256b3.png)

Reply:
I like it. Very spooky. I just hope it doesn't downplay Archon telepathy somehow. Or maybe it should, since this is far from one singular entity.

Reply:
Tests are being funky with an end quote tho.